conda_forge_tick/status_report.py

```diff
@@ -199,6 +205,15 @@ def graph_migrator_status(
             for k in sorted(gx2.successors(node))
             if not gx2[k].get("payload", {}).get("archived", False)
         ]
+        if migrator_name in attrs.get("pre_pr_migrator_status", {}) and node in (
+            out["awaiting-pr"] | out["awaiting-parents"]
```

Comment:
do you need awaiting parents?

Reply:
likely not - I am also wondering if we should just list solvable as another category. Thoughts?

Reply:
I would put not solvable in an independent category. Although the messaging may need to state that not solvable doesn't mean there is a problem, just that the CDN doesn't have the pacakge (or there is an actual problem).

Reply:
We might consider putting the feedstocks that actually error under `bot-error` so we know to look at them more closely (right now I think they just sit as awaiting parents)

Reply:
We could even set the URL to be the run that they most recently errored on

Reply:
not in this pr :)